csharp/Platform.Numbers/Math[T].cs

```diff
@@ -9,20 +9,20 @@
 namespace Platform.Numbers
 {
     /// <summary>
-    /// <para>.</para>
-    /// <para>.</para>
+    /// <para>Represents a set of compiled algebraic operations delegates.</para>
+    /// <para>Представляет набор скомпилированных делегатов алгебраических операци.</para>
     /// </summary>
     public static class Math<T>
     {
         /// <summary>
-        /// <para>.</para>
-        /// <para>.</para>
+        /// <para>A read-only field that represents a taking the absolute value of a number function delegate.</para>
```

Comment:
`a taking the absolute value of a number function delegate` => `a delegate of a function that takes the absolute value of a number`